accounts-cluster-bench/src/main.rs

```diff
@@ -619,6 +902,15 @@ fn main() {
         eprintln!("bad num_instructions: {num_instructions}");
         exit(1);
     }
+    let num_rpc_bench_threads = value_t!(matches, "num_rpc_bench_threads", usize).unwrap_or(0);
+    let rpc_benches = values_t!(matches, "rpc_bench", String)
```

Comment:
I don't think you've actually added these two options to clap?

Reply:
☝️